sbin/firehol

```diff
@@ -253,26 +239,23 @@ fi
 
 # FireHOL config directory.
 # EVEN IF YOU CHANGE THIS, THE firehol-defaults.conf FILE
-# SHOULD STILL EXIST IN THE ORIGINAL $FIREHOL_CONFIG_DIR
-FIREHOL_CONFIG_DIR="$FIREHOL_CONFIG_DIR"
+# SHOULD STILL EXIST IN THE ORIGINAL $SYSCONFDIR/firehol
+FIREHOL_CONFIG_DIR="${FIREHOL_CONFIG_DIR}"
```

Comment:
So, this code is written out to `/etc/firehol/firehol-defaults.conf` when that file does not exist.

I don't think a discussion about how useful that is/isn't belongs to this pull request, so keeping it like this keeps the existing behaviour unchanged even though it looks a little odd.

We can cover the setup of the defaults file elsewhere if you like.